packages/runtime/test/runtime.test.ts

```diff
@@ -141,30 +141,63 @@ describe('runtime', () => {
       }
     `
     await runtime.executeQuery(createPostMutation, {
-      input: { content: { title: 'A first post', text: 'First post content', ranking: 5 } },
+      input: {
+        content: { title: 'A first post', text: 'First post content', ranking: 5 },
+      },
+    })
+    await runtime.executeQuery(createPostMutation, {
+      input: {
+        content: {
+          title: 'A second post',
+          text: 'Second post content',
+          ranking: 4,
+          date: '2023-07-31T12:00:00.000Z',
+        },
+      },
     })
     await runtime.executeQuery(createPostMutation, {
-      input: { content: { title: 'A second post', text: 'Second post content', ranking: 4 } },
+      input: {
+        content: { title: 'A third post', text: 'Third post content', ranking: 7, share: 0.4 },
+      },
     })
 
-    const res = await runtime.executeQuery(
-      `
-      query {
-        viewer {
-          postList(filters: { where: { ranking : { greaterThan: 4 } } }, first: 5) {
-            edges {
-              node {
-                title
-                text
-                ranking
-              }
+    const filterQuery = `query FilterPostQuery($input: PostFiltersInput!) {
+      viewer {
+        postList(filters: $input, first: 5) {
+          edges {
+            node {
+              title
+              text
+              ranking
             }
           }
         }
       }
-      `,
-    )
-    expect(res).toMatchSnapshot()
+    }`
+
+    const filterVars1 = {
+      input: {
+        and: [{ where: { ranking: { greaterThan: 4 } } }, { where: { ranking: { lessThan: 7 } } }],
```

Comment:
```suggestion
        where: { ranking: { greaterThan: 4, lessThan: 7 } },
```
This should be supported as-is.